proto/mls/api/v1/mls.proto

```diff
@@ -163,6 +164,7 @@ message GroupMessageInput {
   message V1 {
     bytes data = 1; // Serialized MlsProtocolMessage
     bytes sender_hmac = 2;
+    optional bool should_push = 4;
```

Comment:
Does making this optional matter? Old messages sent on the old format will just not have this option set.

Reply:
this looks okay to me

Reply:
What is the difference between leaving this option unset and setting it to true?

Seems like the push server would treat them the same

Reply:
should be 3 though not 4